src/core/src/sys/config/particles.rs

```diff
@@ -25,25 +23,31 @@ impl fmt::Display for ParticleKind {
     }
 }
 
-/// The Particle type hold basic data about a particle in the system. It is self
-/// contained, so that it will be easy to send data between parallels processes.
-#[derive(Clone, Debug)]
-pub struct Particle {
-    /// Particle name. This one is not public, as we always want to get &str,
-    /// and to use either `String` of `&str` to set it.
-    name: String,
-    /// Particle kind, an index for potentials lookup
-    pub(in ::sys) kind: ParticleKind,
-    /// Particle mass
-    pub mass: f64,
-    /// Particle charge
-    pub charge: f64,
-    /// Particle positions
-    pub position: Vector3D,
-    /// Particle velocity, if needed
-    pub velocity: Vector3D,
+#[allow(missing_docs)]
+mod derive {
```

Comment:
Is it mandatory to put the struct into an own module here?

Reply:
Not at all, but it is the only way I found to use the `#[allow(missing_docs)]` here. I'll check if it is still necessary.